starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,88 @@
 // Names and Input
+var hacker1 = "John"
+console.log("The driver's name is " + hacker1);
 
+//var hacker2 = prompt("What is your name?")
+var hacker2 = "Luca"
+console.log("The driver's name is " + hacker2);
```

Comment:
You can ask the user to enter a name himself by using "prompt" : 
var hacker2 = prompt("What's your name ? ")